activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fixes issue in `ActiveSupport::Cache::MemCacheStore` which can cause duplicate compression,
+    or cause the provided `compression_threshold` to not be respected.
```

Comment:
The changelog entry is for a bug that's now fixed, let's describe it in the past tense.

```suggestion
*   Fix an issue in `ActiveSupport::Cache::MemCacheStore` which caused duplicate compression,
    and caused the provided `compression_threshold` to not always be respected.
```